src/app/qan/search-autocomplete/search-autocomplete.component.ts

```diff
@@ -24,11 +24,11 @@ export class SearchAutocompleteComponent implements OnInit, OnDestroy {
   ) {
     this.autocomplete$ = this.qanFilterService.getAutocompleteFilters
       .subscribe(configs => {
-        this.autocomplete = configs;
+        this.autocomplete = configs
       });
```

Comment:
The brackets ’{}’ here looks like rudimental.

Reply:
We need it here because we only assign a value here and don't want to return anything